common/artifact-manager/src/main/java/com/vmware/pscoe/iac/artifact/VroPackageStore.java

```diff
@@ -178,8 +178,26 @@ public final List<Package> importAllPackages(final List<Package> vroPackages, fi
 				String backupFilePath = this.createBackupFilePath(pkg, currentDateTimeString, backupFilesDirectory);
 
 				try {
-					pkg.setFilesystemPath(backupFilePath);
-					restClient.exportPackage(pkg, dryrun, exportConfigAttributeValues, exportConfigSecureStringValues);
+					System.out.println("Package for back up: " + pkg.getFQName());
```

Comment:
Can you please user the logger object to send information instead of system.out?